recipes/geoplot/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "geoplot" %}
+{% set version = "0.0.2" %}
+{% set md5 = "4c27bfaf5da2bcb75623bba3a9539ad5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/3a/40/45951c006fdd7f08e33230e7f70809869788f93a7500816b7ce952dfd814/{{ name }}-{{ version }}.tar.gz
+  md5: 4c27bfaf5da2bcb75623bba3a9539ad5
+
+build:
+  skip: True # [win]
+  # Windows install requires resolving https://github.com/SciTools/cartopy/issues/805.
+
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  # number: 1
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - matplotlib
```

Comment:
When using `--single-version-externally-managed` there is a change that all the run time dependencies are not needed at build time. (Although some packages import themselves in the `setup.py` breaking that.)